src/upstream/rss3/mod.rs

```diff
@@ -217,14 +218,51 @@ async fn save_item(p: ResultItem) -> Result<TargetProcessedList, Error> {
         return Ok(vec![]);
     }
 
-    let found = p
-        .actions
-        .into_iter()
-        .find(|a| a.tag == "collectible".to_string());
+    let mut found = None;
+
+    for a in p.actions.iter() {
+        if p.tag == "social" && a.tag_type == "mint" {
+            found = Some(a);
```

Comment:
Will `found` be overwritten by another `iter()` ?

Reply:
found declaim as None before actions iterator, it's a new variable in save item iterator process.

Reply:
Should use `let found = p.actions.iter().find(|a| .....)` instead.

Reply:
good suggestion